### ISSUE TITLE
clinical and annotation data

### ISSUE THREAD

**Issue by roayaei** [PierreBSC/Verona_COVID19#1]
Hi, in the source codes you are using "Clinical_data.txt",  "Immune_suppression.txt" and "Patient_annotation.txt" files which I was not able to find in the supplementary data of the paper. Would you please, if possible, direct me to or share with me that data? Thanks

**Comment by PierreBSC** [comment]
Hi,

Thanks for your interest this work. It seems that we forgot to put this files on GEO... It is possible to reconstruct them from the different raw tables provided on the paper but the best is that I put it on the Github directly. Tell me if it works for you !

**Comment by roayaei** [comment]
Yep, thanks a lot :)

**Comment by roayaei** [comment]
Sorry to bug you again. I believe these files are also missing: "ICU_patient_data.txt", "VT_analysis.txt", "Correspondance_ENSEMBL.txt", "Immuno_globulin_data.txt", "FACS_CD14_monocytes.txt", "Cytokine_monocytes_CD14.txt". I would appreciate it if you could share them with me as well :)

**Comment by roayaei** [comment]
and I suppose I get the same clusters by running your code, right?

**Comment by PierreBSC** [comment]
Hi Abbas,

So basically all the data are available as raw data from the Nature Communications website but I agree this is not convenient. I will try to put all of them on the Github but without guarantee ^^

**Comment by PierreBSC** [comment]
> and I suppose I get the same clusters by running your code, right?

You mean the clustering of the scRNA-seq data ?

**Comment by roayaei** [comment]
Yes, the cell clusters. I would like to replicate your results. Thanks again :)

**Comment by roayaei** [comment]
Sorry to bug you again but could you please add the patient_id column to your immune_suppression data. I couldn't figure it out. Thank you.
